charts/opensearch/templates/statefulset.yaml

```diff
@@ -137,70 +140,70 @@ spec:
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
       volumes:
-      {{- range .Values.secretMounts }}
-      - name: {{ .name | required "secretMount .name is required" }}
-        secret:
-          secretName: {{ .secretName | required "secretMount .secretName is required" }}
-          {{- if .defaultMode }}
-          defaultMode: {{ .defaultMode }}
-          {{- end }}
-      {{- end }}
-      {{- if .Values.config }}
-      - name: config
-        configMap:
-          name: {{ template "opensearch.uname" . }}-config
-      {{- end }}
-      {{- if and .Values.securityConfig.config.securityConfigSecret .Values.securityConfig.config.data }}
-      - name: security-config
-        secret:
-         secretName: {{ .Values.securityConfig.config.securityConfigSecret }}
-      {{- end }}
-      {{- if .Values.securityConfig.actionGroupsSecret }}
-      - name: action-groups
-        secret:
-          secretName: {{ .Values.securityConfig.actionGroupsSecret }}
-      {{- end }}
-      {{- if .Values.securityConfig.configSecret }}
-      - name: security-config
-        secret:
-          secretName: {{ .Values.securityConfig.configSecret }}
-      {{- end }}
-      {{- if .Values.securityConfig.internalUsersSecret }}
-      - name: internal-users-config
-        secret:
-          secretName: {{ .Values.securityConfig.internalUsersSecret }}
-      {{- end }}
-      {{- if .Values.securityConfig.rolesSecret }}
-      - name: roles
-        secret:
-          secretName: {{ .Values.securityConfig.rolesSecret }}
-      {{- end }}
-      {{- if .Values.securityConfig.rolesMappingSecret }}
-      - name: role-mapping
-        secret:
-          secretName: {{ .Values.securityConfig.rolesMappingSecret }}
-      {{- end -}}
-      {{- if .Values.securityConfig.tenantsSecret }}
-      - name: tenants
-        secret:
-          secretName: {{ .Values.securityConfig.tenantsSecret }}
-      {{- end }}
+        {{- range .Values.secretMounts }}
+        - name: {{ .name }}
+          secret:
+            secretName: {{ .secretName }}
+            {{- if .defaultMode }}
+            defaultMode: {{ .defaultMode }}
+            {{- end }}
+        {{- end }}
```

Comment:
You've overwritten some master stuff in your merge. Please carefully re-add this.

Reply:
I have removed the other range over secrets mounts as it has less options.